uvicorn/middleware/message_logger.py

```diff
@@ -39,23 +40,27 @@ def __init__(self, app):
             message = await receive()
             logged_message = message_with_placeholders(message)
             log_text = "%s - ASGI [%d] Sent %s"
-            self.logger.debug(log_text, client_addr, task_counter, logged_message)
+            self.access_logger.debug(
+                log_text, client_addr, task_counter, logged_message
+            )
             return message
 
         async def inner_send(message):
             logged_message = message_with_placeholders(message)
             log_text = "%s - ASGI [%d] Received %s"
-            self.logger.debug(log_text, client_addr, task_counter, logged_message)
+            self.access_logger.debug(
+                log_text, client_addr, task_counter, logged_message
+            )
             await send(message)
 
         log_text = "%s - ASGI [%d] Started"
-        self.logger.debug(log_text, client_addr, task_counter)
+        self.access_logger.debug(log_text, client_addr, task_counter)
```

Comment:
I *think* we probably want to reserve the access logger strictly for `startup/shutdown/request-response`, and put this kind of debug logging on the error logger. (?)

Reply:
not sure I understand what you mean by`startup/shutdown/request-response`, is it in the context of this module or generally speaking ?.

Currently there are currently 5 message in this module, 4 are on access

1. `"%s - ASGI [%d] Sent %s"`
2. `"%s - ASGI [%d] Received %s"`
3. `"%s - ASGI [%d] Started"`
4. `"%s - ASGI [%d] Completed"`

the last one is on error_logger `"%s - ASGI [%d] Raised exception"`

From your comment I'm not sure if
A. you want all of them to be on the error_logger because it's meant for debugging logging
B. something else